cardano-api/src/Cardano/Api/IPC.hs

```diff
@@ -137,10 +139,17 @@ data LocalNodeClientProtocols block point tip tx txerr query m =
          :: Maybe (LocalStateQueryClient   block point query m ())
      }
 
-data LocalChainSyncClient block point tip m
-  = NoLocalChainSyncClient
-  | LocalChainSyncClientPipelined (ChainSyncClientPipelined block point tip   m ())
-  | LocalChainSyncClient          (ChainSyncClient          block point tip   m ())
+data LocalChainSyncClient block point tip m =
+    NoLocalChainSyncClient
+
+  | LocalChainSyncClientPipelined
+     (ChainSyncClientPipelined block point tip m ())
+
+  | LocalChainSyncClient
+     (ChainSyncClient block point tip m ())
+
+  | LocalChainSyncChairman
+     (ChainSyncClient block point tip m ())
```

Comment:
In a follow up PR I want to wrap `ChainSyncClient` in a newtype to make it a bit easier to read and understand what's happening.

Reply:
Humm not sure I understand what that has to do with adding a new constructor here. It seems wrong that that cardano-api is aware of chairman since chairman just seems like a special case of a chain sync client. Am I missing something?

Reply:
You aren't. It's a readability change but I understand you're point.